lib/std/math.zig

```diff
@@ -294,8 +293,13 @@ pub fn floatMantissaBits(comptime T: type) comptime_int {
     };
 }
 
-/// Returns the number of bits in the exponent of floating point type
-/// T.
+/// Returns the number of bits in the mantissa of floating point type T,
+/// including the implied leading 1 or 0.
+pub fn floatMantissaDigits(comptime T: type) comptime_int {
+    return floatMantissaBits(T) + 1;
+}
+
+/// Returns the number of bits in the exponent of floating point type T.
 pub fn floatExponentBits(comptime T: type) comptime_int {
```

Comment:
It would not test my changes but it sounds like the only way to reason about "testing" the magic constants in the functions the others derive from - will do